proxy/proxy.go

```diff
@@ -523,6 +528,23 @@ func callWeaveAttach(container *docker.Container, args []string) error {
 	return nil
 }
 
+func validateCIDRs(cidrs []string) error {
+	for _, cidr := range cidrs {
+		if cidr == "net:default" {
+			continue
+		}
+		for _, prefix := range []string{"ip:", "net:", ""} {
+			if strings.HasPrefix(cidr, prefix) {
+				if _, err := address.ParseCIDR(strings.TrimPrefix(cidr, prefix)); err == nil {
+					break
+				}
+				return fmt.Errorf("invalid CIDR: %s", cidr)
```

Comment:
some extra context would help. "invalid WEAVE_CIDR"? (yes, I know the value could be just one of several elements, but I doubt that would cause confusion).